fonduer/learning/disc_models/rnn/lstm.py

```diff
@@ -0,0 +1,226 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from scipy.sparse import issparse
+
+from fonduer.learning.disc_learning import NoiseAwareModel
+from fonduer.learning.disc_models.rnn.config import get_config
```

Comment:
Probably should use utils config. See previous comment.

Reply:
done!